src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,21 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import mate.academy.spring.validator.Email;
+import mate.academy.spring.validator.Password;
+
+@Password
 public class UserRequestDto {
+    @Email
+    @NotNull
     private String email;
+    @NotNull
+    @Min(6)
     private String password;
+    @NotNull
+    @Min(6)
+    private String repeatPassword;
```

Comment:
All password checks should be inside @Password annotation
```suggestion
    private String password;
    private String repeatPassword;
```
![image](https://user-images.githubusercontent.com/20326858/234840340-d911d803-e7da-40bc-946e-0a92a8758aa8.png)